lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -602,3 +603,11 @@ func getNetworkProbeConfig(networkID string) ([]*lte_mconfig.NProbeTask, *lte_mc
 	}
 	return npTasks, liUes
 }
+
+// getSyncInterval takes network-wide subscriber_db sync interval and overrides it if also set for gateway
+func getSyncInterval(nwEpc *lte_models.NetworkEpcConfigs, gwEpc *lte_models.GatewayEpcConfigs) uint32 {
+	if gwEpc.SubscriberdbSyncInterval != 0 {
+		return gwEpc.SubscriberdbSyncInterval
+	}
+	return nwEpc.SubscriberdbSyncInterval
```

Comment:
What if both aren't set? We need some true default. Pull the true default from a cloud service config, and set that default value to 5m

Reply:
Still need to handle this need for a "true default" sync interval -- there should be some default, and operators shouldn't have to set it for it to take effect

Reply:
I've been thinking about this. A default can also be set on the gateway (in `gateway.mconfig`) and orc8r may not send anything. If we want to definitely send something from the orc8r, I haven't figured out where I should set that default value. I could put it right here in this function, but wonder if there's a more suitable place.

Reply:
Could also have it in `lte/cloud/go/services/lte/obsidian/models/defaults.go`?

Reply:
Placed default config in `lte/cloud/configs/subscriberdb.yml`